modules/c3-assisted-installer-preparing-image-generating.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_oci/installing-c3-assisted-installer.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="c3-assisted-installer-preparing-image-generating_{context}"]
+= Generating the image in the {ai-full}
+
+Create a cluster and download the discovery ISO image.
+
+.Procedure
+
+. Log in to link:https://console.redhat.com/[{ai-full} web console] with your credentials.
+
+. In the *Red Hat OpenShift* tile, select *OpenShift*. 
+
+. In the *Red Hat {product-title}* tile, select *Create Cluster*.
+
+. On the *Cluster Type* page, scroll to the end of the *Cloud* tab, and select *Oracle Cloud Infrastructure (virtual machines)*. 
+
+. On the *Create an OpenShift Cluster* page, select the *Interactive* tile.
+
+. On the *Cluster Details* page, complete the following fields:
++
+[cols="1,3",options="header",subs="quotes"]
+|===
+|Field |Action required
+
+|*Cluster name*
+|Specify the name of your {product-title} cluster. This name is the same name you used to create the resource via the Terraform scripts. The name must be between 1-54 characters. It can use lowercase alphanumeric characters or hyphen (-), but must start and end with a lowercase letter or a number.
+
+|*Base domain*
+|Specify the base domain of the cluster. This is the value used for the `zone_dns` variables in the Terraform scripts that run on {oci-c3-short}. Make a note of the value.
```

Comment:
Totally a suggestion, but if you're supposed to note the value for later use then maybe it would be helpful to specify the reason why you're noting it:
```suggestion
|Specify the base domain of the cluster. This is the value used for the `zone_dns` variables in the Terraform scripts that run on {oci-c3-short}. Make a note of the value for later use.
```